### ISSUE TITLE
wkt property has a keyword argument.

### ISSUE THREAD

**Issue by ShigeruNakagaki** [Toblerity/Shapely#581]
[shaply.geometry.base.wkt](https://github.com/Toblerity/Shapely/blob/bed28170140429d58e4079c57cc2480c5e5ff6ab/shapely/geometry/base.py#L365) property has a keyword argument, but we can not call a property with argument, right?
```
    @property
    def wkt(self, **kw):
```

**Comment by sgillies** [comment]
Correct. The keyword arguments look like they are left over from another time and can be removed.
